internal/provisioner/bundledeployment/bundledeployment.go

```diff
@@ -255,6 +258,33 @@ func (p *bundledeploymentProvisioner) reconcile(ctx context.Context, bd *rukpakv
 		})
 		return ctrl.Result{}, err
 	}
+	if len(chrt.CRDObjects()) > 0 {
+		// Install the CRDs ourselves before installing the rest of the helm chart
+		// This allows us to work around helm's limitations around updating CRDs
+		for _, crdFile := range chrt.CRDObjects() {
+			var c apiextensionsv1.CustomResourceDefinition
+			err := yaml.Unmarshal(crdFile.File.Data, &c)
+			if err != nil {
+				meta.SetStatusCondition(&bd.Status.Conditions, metav1.Condition{
+					Type:    rukpakv1alpha1.TypeInstalled,
+					Status:  metav1.ConditionFalse,
+					Reason:  rukpakv1alpha1.ReasonInstallFailed,
+					Message: err.Error(),
+				})
+				return ctrl.Result{}, err
+			}
+			err = crd.CreateOrUpdateCRD(ctx, p.cl, &c)
```

Comment:
Seeing test failures presumably from failing to set the controller reference on the CRDs - will add.

Reply:
There's a controller-runtime utility that makes this easy. https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.13.1/pkg/controller/controllerutil#SetControllerReference